packages/sanity/src/core/form/inputs/PortableText/object/modals/PopoverModal.tsx

```diff
@@ -79,27 +79,25 @@ function Content(props: PopoverEditDialogProps) {
   const [contentElement, setContentElement] = useState<HTMLDivElement | null>(null)
 
   return (
-    <BoundaryElementProvider element={contentElement}>
-      <ContentContainer width={width}>
-        <ModalWrapper direction="column" flex={1}>
-          <ContentHeaderBox padding={1}>
-            <Flex align="center">
-              <Box flex={1} padding={2}>
-                <Text weight="semibold">{title}</Text>
-              </Box>
+    <ContentContainer width={width}>
+      <ModalWrapper direction="column" flex={1}>
+        <ContentHeaderBox padding={1}>
+          <Flex align="center">
+            <Box flex={1} padding={2}>
+              <Text weight="semibold">{title}</Text>
+            </Box>
 
-              <Button icon={CloseIcon} mode="bleed" onClick={onClose} padding={2} />
-            </Flex>
-          </ContentHeaderBox>
-          <ContentScrollerBox flex={1}>
-            <PresenceOverlay margins={[0, 0, 1, 0]}>
-              <Box padding={3} ref={setContentElement}>
-                {props.children}
-              </Box>
-            </PresenceOverlay>
-          </ContentScrollerBox>
-        </ModalWrapper>
-      </ContentContainer>
-    </BoundaryElementProvider>
+            <Button icon={CloseIcon} mode="bleed" onClick={onClose} padding={2} />
+          </Flex>
+        </ContentHeaderBox>
+        <ContentScrollerBox flex={1}>
+          <PresenceOverlay margins={[0, 0, 1, 0]}>
```

Comment:
Should `<VirtualizerScrollInstanceProvider scrollElement={contentElement}>` also be here?

Reply:
ah yes missed that, will push that change now